src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -34,9 +34,9 @@ const BurgerMenu = () => {
   return (
     <Sheet>
       <SheetTrigger>
-        <Menu color="white" className="ml-2 mr-2" />
+        <Menu color="white" className="ml-2 mr-2 " />
```

Comment:
Tem um espaço sobrando no atributo de classe

```suggestion
        <Menu color="white" className="ml-2 mr-2" />
```